### PR TITLE
Example scenario for big data + IoT

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/data/big-data-with-iot.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* [Azure Stream Analytics](/azure/stream-analytics) is an event-processing engine that can analyze high volumes of data streaming from devices and other data sources. It also supports extracting information from data streams to identify patterns and relationships. In this scenario, Stream Analytics ingests and analyzes data from IoT devices and stores the results in Azure SQL Database. 
+* [Azure SQL Database](/azure/sql-database) contains the results of analyzed data from IoT devices and meters, which can be viewed by analysts and users via an Azure-based Web application. 
+* [Azure Blob storage](/azure/storage/blobs) stores image data gathered from the IoT hub devices. The image data can be viewed via the web application.
+* [Traffic Manager](/azure/traffic-manager) controls the distribution of user traffic for service endpoints in different Azure regions.
+* [Load Balancer](/azure/load-balancer) lets you scale your applications and create high availability for your services. The load balancer distributes data submissions from construction equipment devices.
```
Comment (MikeWasson) [inline docs/example-scenario/data/big-data-with-iot.md]: > lets you scale your applications

Could be confusing whether "application" means the whole solution or the app running on the VMs.

Maybe say "scale out VMs" or "distribute load across VMs"? (or: Scale out the web application?)

File: docs/example-scenario/data/big-data-with-iot.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+* [Azure Virtual Machines](/azure/virtual-machines) host the web services that receive and ingest the construction results data into the Apache Cassandra database.
+* [Apache Cassandra](http://cassandra.apache.org/) is a distributed NoSQL database used to store construction data for later processing by Apache Spark.
```
Comment (MikeWasson) [inline docs/example-scenario/data/big-data-with-iot.md]: Any particular reason that Cassandra was used in this scenario (vs CosmosDB?)

File: docs/example-scenario/data/big-data-with-iot.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* [Web Apps](/azure/app-service) hosts the end-user web application, which can be used to query and view source data and images. Users can also initiate batch jobs in Apache Spark via the application.
+* [Apache Spark on HDInsight](/azure/hdinsight/spark) supports in-memory processing to boost the performance of big-data analytic applications. In this scenario, Spark is used to run complex algorithms over the data stored in Apache Cassandra.
+
+### Alternatives
+
```
Comment (MikeWasson) [inline docs/example-scenario/data/big-data-with-iot.md]: Add Data Bricks as an alternative to HDInsight

File: docs/example-scenario/data/big-data-with-iot.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/big-data-with-iot.md]: Fixed.

File: docs/example-scenario/data/big-data-with-iot.md (hunk 3 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/big-data-with-iot.md]: Added.

File: docs/example-scenario/data/big-data-with-iot.md (hunk 2 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/big-data-with-iot.md]: I believe the customer built the solution before Cosmos DB was available.

Comment (MikeWasson) [reply docs/example-scenario/data/big-data-with-iot.md]: fair enough! 😺

**Pull request merged**